webapp/channels/src/i18n/en.json

```diff
@@ -2453,7 +2453,7 @@
   "admin.system_users.filters.role.system_guest": "Guest",
   "admin.system_users.filters.role.system_user": "Member",
   "admin.system_users.filters.role.title": "Role",
-  "admin.system_users.filters.status.active": "Active users",
+  "admin.system_users.filters.status.active": "Activated users",
```

Comment:
I was going to ask if the corresponding `defaultMessage` had to be updated, but it turns out it uses the correct text. I forgot that this file is the source of truth

Reply:
Yeah I think I had forgotten to update this one and not the other one tbh :P